toolkit/tools/imagecustomizerapi/module.go

```diff
@@ -8,14 +8,23 @@ import (
 )
 
 type Module struct {
```

Comment:
This design is a little odd. If you want to set kernel module options, you have to set it to be auto-enabled. Also, the `Options` field does nothing when attached to the `Disable` list.

Perhaps the `Enable` and `Disable` list should just be string lists and then have a separate list for configuring options?